### PR TITLE
 Improving details of booking showed when tapped on calendar

### PR DESCRIPTION
**Pull request by julien4215**
Required for PR https://github.com/aeecleclair/Titan/pull/185

### CODE REVIEW

File: app/endpoints/booking.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ async def get_bookings_for_manager(
 
 @router.get(
     "/booking/bookings/confirmed",
-    response_model=list[schemas_booking.BookingReturn],
+    response_model=list[schemas_booking.BookingReturnApplicant],
```
Comment (armanddidierjean) [inline app/endpoints/booking.py:187]: I'm not sure if we want to return the applicant Phone number and email address

**Review by armanddidierjean — CHANGES_REQUESTED** [review]

File: app/endpoints/booking.py (hunk 1 above)
Comment (julien4215) [reply app/endpoints/booking.py:187]: I didn't think about that, I'll create a new endpoint for managers

Comment (julien4215) [reply app/endpoints/booking.py:187]: Yes I didn't pay attention to that.
I actually don't need to make changes to the API.

**Review by armanddidierjean — APPROVED** [review]

**Review by Rotheem — CHANGES_REQUESTED** [review]
The migration's file needs to be entirely changed

**Review by Rotheem — APPROVED** [review]

**Pull request merged**